controllers/chat/group/removeGroupMember.js

```diff
@@ -31,7 +31,7 @@ const removeGroupMember = async (req, res) => {
       const currentChannel = await client.channel(CHANNEL_TYPE_STRING.GROUP, channelId);
 
       const textOwnUser = `You removed ${targetUserModel.username} from this group`;
-      const textTargetUser = `You has been removed from this group`;
+      const textTargetUser = `${ownUser.username} removed you from this group`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the message text from `You has been removed from this group` to `${ownUser.username} removed you from this group` is a good improvement for clarity and grammar. However, please ensure that `ownUser.username` is always defined and not null to avoid potential issues.
```diff
- const textTargetUser = `You has been removed from this group`;
+ const textTargetUser = `${ownUser.username} removed you from this group`;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->